integration-cli/docker_cli_build_test.go

```diff
@@ -2128,7 +2147,7 @@ func TestBuildRelativeWorkdir(t *testing.T) {
 
 func TestBuildWorkdirWithEnvVariables(t *testing.T) {
 	name := "testbuildworkdirwithenvvariables"
-	expected := "/test1/test2/$MISSING_VAR"
```

Comment:
For reference: this means that we are making a breaking change: undefined variables are now substituted as expected. The rationale for this old behavior was to maintain backwards compatibility when we introduced ENV var substitution. Now that ENV var substitution is expected, it is much more confusing to keep it, since people expect it to be replaced by an empty string.